src/App/Hangouts/Commands.elm

```diff
@@ -18,18 +18,29 @@ getHangouts =
     Http.get hangoutsUrl (decodeCollection hangoutDecoder)
         |> Http.send OnFetchHangouts
 
+
 hangoutDecoder : Decoder Hangout
 hangoutDecoder =
     succeed Hangout
         |: (stringDecoder "date")
-        |: (field "groups" (decodeCollection groupDecoder))
+        |: (groupsDecoder "groups")
```

Comment:
Remembered all `flip` does is interchange the first two arguments. So I gave the flipped `field` arguments in the correct order. 😅